lib/jekyll-mentions.rb

```diff
@@ -8,24 +8,27 @@ class Mentions
     GITHUB_DOT_COM = "https://github.com".freeze
     BODY_START_TAG = "<body".freeze
 
+    OPENING_BODY_TAG_REGEX = %r!<body(.*)>\s*!
```

Comment:
Same questions as https://github.com/jekyll/jemoji/pull/74#discussion_r184102377... cross-posting theme here for ease of reference:

1. Does the `.*` need to be non-greedy? (not sure which is more fault tolerant)
2. Does this need to be case insensitive to allow for `<BODY` tags?
3. Could we simplify/collapse the existing logic with `BODY_START_TAG`?

Reply:
https://github.com/jekyll/jemoji/pull/74#discussion_r184131700